Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml

```diff
@@ -3,1168 +3,1209 @@ commonfields:
   id: MicrosoftGraphMail
   version: -1
 configuration:
-- defaultvalue: https://graph.microsoft.com
```

Comment:
please apply only the relevant changes to avoid this big diff

Reply:
this is a consequence of the plugin use i believe